.changeset/2874.md

```diff
@@ -0,0 +1,12 @@
+---
+'@backstage/plugin-catalog-backend': minor
```

Comment:
i mean obviously it should not be minor, but i just wanted a changelog entry to get in there - is this the right way to do it at the moment while we are in alpha?

Reply:
It should totally be a minor, and yes. Although we're not using this to create releases yet but we should very soon. Let's discuss this asap next week when we're back kinda soon